### PR TITLE
Fixes #24939 - Show id of the report on command line

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by xprazak2** [comment]
Needs https://github.com/theforeman/smart_proxy_openscap/pull/63 to work

File: lib/foreman_scap_client/client.rb [hunk 1]
```diff
@@ -190,5 +192,14 @@ def ensure_tailoring_file
     def download_uri(download_path)
       foreman_proxy_uri + "#{download_path}"
     end
+
+   def foreman_upload_result(response)
+     parsed = JSON.parse(response.body)
+     if parsed['id']
+       puts "Report uploaded, report id: #{parsed['id']}"
+     else
+      puts "Report not uploaded from proxy to Foreman server, cause: #{parsed['result']}"
```
Comment (ares) [inline lib/foreman_scap_client/client.rb]: missing space causing wrong indentation

File: lib/foreman_scap_client/client.rb [hunk 2]
```diff
@@ -190,5 +192,14 @@ def ensure_tailoring_file
     def download_uri(download_path)
       foreman_proxy_uri + "#{download_path}"
     end
+
+   def foreman_upload_result(response)
+     parsed = JSON.parse(response.body)
```
Comment (ares) [inline lib/foreman_scap_client/client.rb]: what if proxy runs older version of smart_proxy_openscap and does not response with valid JSON? should we rescue the parsing error? I'm getting following error on such scenario

```
Uploading results to https://foreman.example.tst:8443/compliance/arf/5
Upload failed: A JSON text must at least contain two octets!
```

**Review by ares — CHANGES_REQUESTED** [review]
two small things, then good to go in

**Comment by xprazak2** [comment]
I rescue the parsing error and print nothing.

**Review by ares — APPROVED** [review]

**Comment by ares** [comment]
Thanks @xprazak2, merging!

**Pull request merged**